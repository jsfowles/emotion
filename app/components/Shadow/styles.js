import glamorous from 'glamorous';

import { remCalc } from '../../lib/styleMethods';

export const Shadow = glamorous.div({
  position: 'absolute',
  width: '100%',
  bottom: `-${remCalc(25)}`,
  left: 0,
  backgroundImage: 'radial-gradient(closest-side, rgba(0,0,0,0.6) 0, rgba(0,0,0,0.2) 50%, transparent 100%)',
  backgroundPosition: `0% -${remCalc(15)}`,
  backgroundRepeat: 'no-repeat',
  height: remCalc(25),
  overflow: 'hidden',
  pointerEvents: 'none',
});
