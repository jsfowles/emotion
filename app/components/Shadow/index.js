import React from 'react';

import { Shadow } from './styles';

export default () => <Shadow />;
