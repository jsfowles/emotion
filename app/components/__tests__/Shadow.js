import React from 'react';
import renderer from 'react-test-renderer';

import Shadow from '../Shadow';

describe('Shadow', () => {
  test('renders', () => {
    const tree = renderer.create(<Shadow />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
