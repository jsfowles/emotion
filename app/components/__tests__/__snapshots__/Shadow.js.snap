// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shadow renders 1`] = `
<div
  className="css-yew1ls"
/>
`;
